plugin.js

```diff
@@ -199,18 +708,18 @@
 						{
 							editor.on('selectionChange', function(ev)
 							{
-								var regexp = new RegExp('^' + placeholdersRegex + '$');
 								var currentValue = this.getValue();
 
 								var elements = ev.data.path.elements;
 								for(var i = 0; i < elements.length; i ++)
 								{
 									var element = elements[i];
-									var text = element.$.innerText.toString();
-									if(text !== currentValue && text.match(regexp))
-									{
-										this.setValue(text, placeholdersMap[text].label);
-										return;
+									if (element.$.innerText !== undefined) {

```

Comment:
Always would be pretty bad :) I'll see if I can reproduce it, ideally this would be fixed in a way that prevents it from happening in the first place.